helpers/discovery/editDiscovery.ts

```diff
@@ -16,7 +21,7 @@ export async function editDiscovery(bot: Bot, guildId: bigint, data: ModifyGuild
 
   return {
     guildId,
-    primaryCategoryId: result.primary_category_id,
+    primaryCategoryId: BigInt(result.primary_category_id),
```

Comment:
Are u sure this is a snowflake and not a categor name?

Reply:
I do not know. These endpoints are not documented.

Reply:
Updated type.